tests/cloudpickle_test.py

```diff
@@ -1588,7 +1588,7 @@ def g():
 
         self.assertEqual(f2.__doc__, f.__doc__)
 
-    @unittest.skipIf(sys.version_info < (3, 7),
+    @unittest.skipIf(sys.version_info < (3, 4),
                      """This syntax won't work on py2 and pickling annotations
                      isn't supported for py36 and below.""")
```

Comment:
The comment should also be updated ("3.6" -> "3.3")

Reply:
fixed